src/containers/Activity/Activity.js

```diff
@@ -57,7 +57,9 @@ export class Activity extends React.PureComponent {
       );
     }
   }
-  componentWillReceiveProps(nextProps) {
+
+  // Renamed via https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
@Brainenjii should we change this to `componentDidUpdate()` and `getDerivedStateFromProps()`?